pkg/apis/triggers/v1alpha1/trigger_template_validation_test.go

```diff
@@ -115,28 +115,6 @@ func TestTriggerTemplate_Validate(t *testing.T) {
 		template *v1alpha1.TriggerTemplate
 		want     *apis.FieldError
 	}{{
-		name: "invalid objectmetadata, name with dot",
```

Comment:
Was this intentional?

Reply:
when i update dependency i see we are not getting below error
```
Invalid resource name: special character . must not be present
```
when there is a `.` in objectmeta name

Reply:
hmmm....that's interesting.

Reply:
Found it: https://github.com/tektoncd/pipeline/issues/3891

We rely on pipelines for object meta validation. And pipelines recently started allowing dots in object names.